docs/documentation/platform/access-controls/abac/managing-machine-identity-attributes.mdx

```diff
@@ -64,17 +64,31 @@ For methods like OIDC, these come as claims in the token and can be made availab
 
   </Tab>
   <Tab title="Kubernetes Login Attributes">
-				For identities authenticated using Kubernetes, the service account's namespace and name are available in their policy and can be accessed as follows:
+		For identities authenticated using Kubernetes, the service account's namespace and name are available in their policy and can be accessed as follows:
 
     ```
     {{ identity.auth.kubernetes.namespace }}
     {{ identity.auth.kubernetes.name }}
     ```
 
     <img src="/images/platform/access-controls/abac-policy-k8s-format.png" />
+  </Tab>
+  <Tab title="AWS Attributes">
+    For identities authenticated using AWS Auth, several attributes can be accessed:
+
```

Comment:
Can you add with an example here like

For  an arn `arn:aws:iam::123456789012:user/MyUserName`

These would be
```
    {{ identity.auth.aws.accountId }} \\ 123456789012
    ...
```